locations/storefinders/algolia.py

```diff
@@ -27,6 +28,18 @@ class AlgoliaSpider(Spider):
     index_name: str = ""
     referer: str | None = None
```

Comment:
Does `referer` need to be extracted too via `DetectionRequestRule(headers=...)` or can this be left as None in most cases?

Reply:
Thanks for confirming! :)